schedule/functional/minimal_x.yaml

```diff
@@ -1,16 +1,90 @@
+---
 name:           minimal_x
 description:    >
-    Maintainer: slindomansilla
-    As default but explicitly unselecting DE patterns, for example gnome, to have only a "minimal X" environment preserved, for example icewm.
+    Maintainer: qe-core <qe-core@suse.de>
+    As default but explicitly unselecting DE patterns,
+    for example gnome, to have only a "minimal X" environment preserved,
+    for example icewm.
+
+conditional_schedule:
+    boot:
+        ARCH:
+            x86_64:
+                - installation/isosize
+                - '{{bootloader}}'
+            s390x:
+                - installation/bootloader_zkvm
+            aarch64:
+                - installation/isosize
+                - '{{bootloader}}'
+            ppc64le:
+                - installation/isosize
+                - installation/bootloader
+    bootloader:
+        MACHINE:
+            64bit:
+                - installation/bootloader
+            aarch64:
+                - installation/bootloader_uefi
+            uefi:
+                - installation/bootloader_uefi
+    encrypt_lvm:
+        MACHINE:
+            64bit:
+                - '{{lvm}}'
+            aarch64:
+                - '{{lvm}}'
+            ppc64le:
+                - '{{lvm}}'
+    lvm:
+        LVM:
+            1:
+                - installation/partitioning/encrypt_lvm
+    mgmt_console:
+        ARCH:
+            s390x:
+                - boot/reconnect_mgmt_console
+    grub_test:
+        ARCH:
+            x86_64:
+                - installation/grub_test
+            aarch64:
+                - installation/grub_test
+            ppc64le:
+                - installation/grub_test
+    system_workarounds:
+        ARCH:
+            aarch64:
+                - installation/system_workarounds
+    glibc_sanity:
+        ARCH:
+            x86_64:
+                - console/glibc_sanity
+    x_vt:
+        ARCH:
+            x86_64:
+                - console/x_vt
+            aarch64:
+                - console/x_vt
+            ppc64le:
+                - console/x_vt
+    ncurses:
```

Comment:
Was there a reason as to why not on S390?`

Reply:
From main_common :
`    if (get_var("DESKTOP") !~ /textmode/ && !is_s390x) {
        loadtest "console/x_vt";
    }
`
again, I have no clue about the history of this, so +1 for reviewing